src/components/form-datepicker/form-datepicker.js

```diff
@@ -379,7 +379,9 @@ export const BFormDatepicker = /*#__PURE__*/ Vue.extend({
     const idWrapper = this.safeId('_b-form-date_')
 
     let $button = h('div', { attrs: { 'aria-hidden': 'true' } }, [
-      this.isHovered || this.hasFocus
+      this.$scopedSlots['button-content']
```

Comment:
Please make sure to use `this.normalizeSlot()` here.
For that you have to include the `normalizeSlot` mixin.

Reply:
The normalize slot mixin can be imported from `utils/normalize-slot`

Reply:
For the scope, we may want to add in the value of `this.state` so that the button content could optionally be styled based on the validation state.

Reply:
I have added in the normalize-slot mixin, and made a few tweaks.